manifests/rhoai/addon/apps/starburst/starburst-galaxy-quickstart.yaml

```diff
@@ -62,7 +62,7 @@ spec:
         ### To launch a Jupyter workbench from within the Red Hat OpenShift AI console:
         1. Click **Applications** &#x2192; **Enabled**.
         2. Find the **Jupyter** card.
-        3. Click **Launch** on the Jupyter card to access the Jupyter **Start a workbench** page.  A new browser tab will open displaying the **Start a workbench** page.
+        3. Click **Launch** on the Jupyter card to access the Jupyter **Start a basic workbench** page.  A new browser tab will open displaying the **Start a workbench** page.
```

Comment:
Same here, change it to "Launch application" and change the second "Start a workbench" to "Start a basic workbench". Also, are there 2 spaces between the first and second sentences?